### PR TITLE
Merchant Employee - Items CRUD (US 42 - 48)

### PR DESCRIPTION
**Pull request by javier-aguilar**
- Merchant employee can activate/deactivate an item
- Merchant employee can delete an item
- Merchant employee can add an item with valid information (no missing fields except for image)
- Default image placeholder when Merchant employee adds an item
- Merchant employee can edit an item with valid information  (no missing fields except for image)

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Item <ApplicationRecord
+  after_initialize :set_defaults
```
Comment (aperezsantos) [inline app/models/item.rb]: Not sure how important this is, but this might look cleaner with different describe blocks for validations/relationships.

Comment (aperezsantos) [reply app/models/item.rb]: My mistake. I thought I was looking at the spec file

File: app/views/merchant/dashboard/index.html.erb [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
 <h1>Merchant Dashboard</h1>
 
-<% if defined? @current_user.merchant %>
-  <%= @current_user.merchant.name %>
-  <%= @current_user.merchant.address %>
-  <%= @current_user.merchant.city %>
-  <%= @current_user.merchant.state %>
-  <%= @current_user.merchant.zip %>
+  <center>
+  <%= link_to "View Merchant Items", "/merchant/items" %>
```
Comment (aperezsantos) [inline app/views/merchant/dashboard/index.html.erb]: Indentation needed, but other than that, it looks great.

File: config/routes.rb [hunk 3]
```diff
@@ -1,13 +1,7 @@
...
   root "welcome#index"
 
-  get "/merchants", to: "merchants#index"
-  get "/merchants/new", to: "merchants#new"
-  get "/merchants/:id", to: "merchants#show"
-  post "/merchants", to: "merchants#create"
-  get "/merchants/:id/edit", to: "merchants#edit"
-  patch "/merchants/:id", to: "merchants#update"
-  delete "/merchants/:id", to: "merchants#destroy"
+  resources :merchants
```
Comment (aperezsantos) [inline config/routes.rb:4]: Awesome!

**Review by aperezsantos — APPROVED** [review]
Some spacing improvements can be made, but overall, things look great. Thank you!

**Review by reid-andrew — APPROVED** [review]

**Pull request merged**